src/Symfony/Contracts/HttpClient/Test/HttpClientTestCase.php

```diff
@@ -795,7 +795,7 @@ public function testMaxDuration()
 
         $duration = microtime(true) - $start;
 
-        $this->assertGreaterThanOrEqual(0.1, $duration);
+        $this->assertGreaterThanOrEqual(0.098, $duration);
```

Comment:
> Failed asserting that 1.3819499015808 is less than 0.2.

in https://ci.appveyor.com/project/fabpot/symfony/builds/26500970

Reply:
Hmm not sure if we can test the duration then. Pretty sure this case happens because it tooks 1.38sec until STREAM_NOTIFY_CONNECT. With the native http client, the actual duration can be really bigger than the configured max duration since we rely on the stream notifications.

Reply:
just remove that assertion... `/max-duration` never stops (let's call `set_time_limit(0)` in there btw)
and replace the try/catch above with an `expectException`, that will prove it works